sensors/nxt_i2c_sensor_defs.c

```diff
@@ -25,6 +25,36 @@
 #include "ms_ev3_smux.h"
 #include "ms_nxtmmx.h"
 
+/*
+ * convert raw data of LEGO NXT Temperature Sensor to Celsius
+ */
+static int lego_nxt_temp_celsius_scale(void *context,
```

Comment:
The lego-sensor class has linear scaling built in, so we shouldn't need any special functions like this unless scaling is non-linear.